bin/condor_submit

```diff
@@ -63,21 +64,101 @@ def main():
         - get credentials
         - handle tarfile options
         - set added values from environment, etc.
-        - convert/render template files to submission files
         - launch
     """
-    cmd_args = sys.argv[1:]
 
-    proxy, token = get_creds(args)
+    condor_parser = argparse.ArgumentParser(
+        description='Queue jobs for execution under HTCondor without the jobsub wrapper.')
+
+    condor_parser.add_argument(
+        "-G",
+        "--group",
+        required=False,
+        help="Group/Experiment/Subgroup for priorities and accounting",
+        action=StoreGroupinEnvironment)
+
+    condor_parser.add_argument(
+        "-name",
+        required=False,
+        help="Submit to the specified condor_schedd")
+
+    condor_parser.add_argument(
+        "-spool",
+        required=False,
+        help="Spool all required input files, job event log, and proxy over the connection to the condor_schedd",
+        action='store_true')
+
+    condor_parser.add_argument(
+        "-remote",
+        required=False,
+        help="Submit to the specified condor_schedd, spooling all required input files over the network connection")
+
+    condor_parser.add_argument(
+        "--credentials",
+        help="credential information")
+
+
+#   PARSE ARGS TO GET SUBMISSION ARGUMENTS
+
+    args_tuple = condor_parser.parse_known_args()
+
+    if args_tuple[0].group is not None:
+        group = args_tuple[0].group
+    else:
+        group = os.environ["GROUP"]
+
+    if args_tuple[0].name is not None:
+        schedd_name = args_tuple[0].name
+    elif args_tuple[0].remote is not None:
+        schedd_name = args_tuple[0].remote
+    else:
+        schedd_add = get_schedd({"group": group})
+        schedd_name = schedd_add.eval("Machine")
+    
+    cmd_args = args_tuple[1]
+
+    print("schedd_name is : %s" % schedd_name)
+    print("cmd_args is : %s" % cmd_args)
+
+
+    if args_tuple[0].spool:
+       schedd_args = "-name '%s' -spool" %(schedd_name)
+    else:
+       schedd_args = "-remote '%s'" %(schedd_name)
+
+
+#   GET CREDENTIALS
+#   these args may/may not come from the input args. for now let's keep it empty  
+    cred_args = {}
+    if args_tuple[0].credentials is not None:
+        cred_args = args_tuple[0].credentials
+
+    proxy, token = get_creds(cred_args)   
+    print("proxy is : %s" % proxy)
+    print("token is : %s" % token)
+
+# put in environment for condor libs to use
+    os.environ["X509_USER_PROXY"] = proxy
+    os.environ["BEARER_TOKEN_FILE"] = token
+
 
-    # put in environment for condor libs to use
-    os.environ['X509_USER_PROXY'] = proxy
-    os.environ['BEARER_TOKEN_FILE'] = token
+#   CREATE SUBMIT COMMAND
```

Comment:
Is there any reason we need to replicate the code from `condor.py:submit()` here rather than using the `submit` call already present?  I'm loathe to have two different ways we submit jobs.

Would it be possible to, for example, pass the information you've parsed using vargs to `submit`, and then make hopefully slight modifications to the `submit` function itself to support the flags you've added support for here?  Let me know if that doesn't make sense.

Reply:
I will look into it. I felt I was going to have a bunch of "if condor_submit, do this, else if jobsub_submit do this other thing". Is that ok?

Reply:
How many of these kinds of conditionals would you think you need to add to `condor.py:submit()`?  I'd hope it's only a couple of differences, if any, between the two.  And since the signature for that `submit()` function only cares about sets of args (`vargs`, `cmd_args`), I'm hoping not too much will change.  Please let me know if that's not the case.

Reply:
I made the changes. I like the result.